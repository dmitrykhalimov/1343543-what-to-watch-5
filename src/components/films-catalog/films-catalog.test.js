import React from "react";
import renderer from "react-test-renderer";
import FilmsCatalog from "./films-catalog";

const MockComponent = () => {
  return (
    <div>
    </div>
  );
};

describe(`FilmsCatalog`, () => {
  it(`Should FilmsCatalog render correctly`, () => {
    const tree = renderer
      .create(
          <FilmsCatalog>
            <MockComponent/>
          </FilmsCatalog>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
