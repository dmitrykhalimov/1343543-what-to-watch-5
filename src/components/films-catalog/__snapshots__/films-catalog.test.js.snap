// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmsCatalog Should FilmsCatalog render correctly 1`] = `
<section
  className="catalog"
>
  <h2
    className="catalog__title visually-hidden"
  >
    Catalog
  </h2>
  <div />
</section>
`;
